psi4/driver/frac.py

```diff
@@ -421,8 +512,19 @@ def ip_fitting(molecule, omega_l, omega_r, **kwargs):
         read = True
         read180 = kwargs['read']
 
-    # The molecule is required, and should be the neutral species
+    # Make sure the molecule the user provided is the active one, and neutral
+    molecule = kwargs.pop('molecule', core.get_active_molecule())
+    molecule.update_geometry()
+
+    if molecule.molecular_charge() != 0:
+        raise ValidationError("""IP Fitting requires neutral molecule to start.""")
+    if molecule.schoenflies_symbol() != 'c1':
```

Comment:
Sorry if I missed it, but should there be an analogous check for UHF reference, too?  Perhaps with an automagic switch from RHF if needed.

Reply:
done! for ip_fitting at least. frac_traverse seems to have a role for HF refs.